perf/benchmark/security/generate_policies/README.md

```diff
@@ -293,11 +356,48 @@ go run generate_policies.go generate.go -configFile="config.json" > authZPolicy.
 ```
 
 ```bash
-go run generate_policies.go generate.go -generate_policy="PeerAuthentication:1,mtlsMode:DISABLE"
+go run generate_policies.go generate.go -configFile="config.json" > peerAuthN.yaml
 ```
 
 - This creates 1 PeerAuthentication policy which has the mtls mode set to DISABLE
 
+## Example 4
+
+- To run a perf test with a requestAuthentication and an AuthorizationPolicy, create a config file called config.json and run the following command:
+
+```json
+{
+  "authZ":
+  {
+    "action":"ALLOW",
+    "numPolicies":1,
+    "numRequestPrincipals":1,
+    "matchRequestPrincipal":true
+  },
+  "requestAuthN":
+  {
+    "numPolicies":1,
+    "NumJwks":1
+  }
+}
+```
+
+```bash
+go run generate_policies.go generate.go -configFile="config.json" > requestAuthN.yaml
+```
+
+- This creates 1 AuthorizationPolicy which has a requestPrincipals rule which will match to the JWKS which is created in the RequestAuthentication policy. This command also
+creates a file called token.txt which is signed with the corresponding public key to match with the JWKS. To use this token in the perf test we must save the token in an environment variable and pass that in as a header to the perf test.
```

Comment:
a file called token.txt which is the JWT token signed with ...

Reply:
thanks